src/_locales/te/messages.json

```diff
@@ -0,0 +1,477 @@
+  },
+  "optionEcLow": {
+    "message": "తక్కువ (7%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "6e3061544eb22cab11cd1222788610bd"
+  },
+  "optionEcMedium": {
+    "message": "మధ్యస్థం (15%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "0ea96235848be6f66788cad72bc0ea02"
+  },
+  "optionEcQuartile": {
+    "message": "క్వార్టైల్ (25%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "344e853acdaef0ac95e161cdbc201036"
+  },
+  "optionEcHigh": {
+    "message": "అధిక (30%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "30035374d52e135473d464fe1726ad8c"
+  },
+  "optionErrorCorrectionDescr": {
+    "message": "ఈ విలువ కోడ్‌లో ఎంత తప్పిపోయిందో సూచిస్తుంది, తద్వారా అది ఇప్పటికీ స్కాన్ చేయబడవచ్చు.",
+    "description": "The description of the error correction option.",
+    "hash": "2bbbd8aafc044a910d26cbf04e560766"
+  },
+  "optionErrorCorrectionDescrLink": {
+    "message": "https://en.wikipedia.org/wiki/QR_code#Error_correction",
+    "description": "The 'Learn more' link in the description of the error correction option.",
+    "hash": "530f6b1ee177dddd7d40571d98a64408"
+  },
+  "optionAutoGetSelectedText": {
+    "message": "వెబ్‌సైట్‌లో ఎంచుకున్న వచనాన్ని స్వయంచాలకంగా ఉపయోగించండి",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "21667c5a30f89561cea02351172b9c91"
+  },
+  "optionUseMonospaceFont": {
+    "message": "మోనోస్పేస్ ఫాంట్ ఉపయోగించండి",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "69191273c436064be8c4f5d6a8f675f5"
+  },
+  "optionDebugMode": {
+    "message": "డీబగ్ మోడ్‌ని ప్రారంభించండి",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "de6506af4973e0980d64f242fba11a19"
+  },
+  "optionDebugModeDescr": {
+    "message": "బగ్‌లు మొదలైనవాటిని నివేదించడానికి కన్సోల్‌లో మరింత వివరణాత్మక లాగింగ్ అవుట్‌పుట్ పొందడానికి మాత్రమే ఇది ఉపయోగపడుతుంది.",
+    "description": "The description shown for the debug mode (optionDebugMode).",
+    "hash": "2aff10770796694827ec4253860e0771"
+  },
+  "translatorCredit": {
+    "message": "This add-on has been translated into English by $TRANSLATORS$.",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
```

Comment:
You can credit yourself here and provide a link (in `translatorLink`) and a name (in `translatorUsername`). Also, you can add yourself to the `CONTRIBUTORS.md` file and the changelog as a translator to [give credits here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit).

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

Reply:
I think you meant @devulapallisai 🙃